nf_core/modules/pipeline_modules.py

```diff
@@ -317,3 +327,33 @@ def has_valid_pipeline(self):
         nf_config = os.path.join(self.pipeline_dir, "nextflow.config")
         if not os.path.exists(main_nf) and not os.path.exists(nf_config):
             raise UserWarning(f"Could not find a 'main.nf' or 'nextflow.config' file in '{self.pipeline_dir}'")
+        return True
+
+    def has_modules_file(self):
+        """Checks whether a module.json file has been created and creates one if it is missing"""
+        modules_json = os.path.join(self.pipeline_dir, "modules.json")
+        if not os.path.exists(modules_json):
```

Comment:
```suggestion
        modules_json_path = os.path.join(self.pipeline_dir, "modules.json")
        if not os.path.exists(modules_json_path):
```

This way you only have to specify it once